src/components/DpInput/DpInput.vue

```diff
@@ -2,6 +2,7 @@
   <div :class="containerWidth !== '' ? prefixClass(containerWidth) : false">
     <dp-label
       v-if="label.text !== ''"
+      :class="{ 'hide-visually': label.hidden }"
```

Comment:
class should be below v-bind. Not only for the alphabetic order, but also because with vue3 there will be a change in the handling of the order:
https://v3-migration.vuejs.org/breaking-changes/v-bind#_3-x-syntax